### ISSUE TITLE
Add a paint gun deactivation field to aperture tag coop checkpoints.

### ISSUE THREAD

**Issue by FrozenDragon0** [BEEmod/BEE2-items#1879]
![20170423201144_1](https://cloud.githubusercontent.com/assets/18079316/25313445/5e1b1b6c-2861-11e7-8a0a-e84d9ffdb4aa.jpg)

**Comment by TeamSpen210** [comment]
Good idea! I think it'd go just after the entry door.

**Comment by TeamSpen210** [comment]
Perhaps it could be a magic fizzler which restores your gun when you return.

**Comment by ZombiePwner1** [comment]
@TeamSpen210 Indeed. Also there is a bug where atlas's paint gun doesn't reset when he dies from a checkpoint and he's paintgun will remain active you need add a trigger to deactivate the paintgun.

**Comment by IceTudor** [comment]
Uhh, quick question. Do I need AP-TAG installed? because neither the field or the gun is showing to me.

**Comment by Endermage77** [comment]
Yes.

**Comment by IceTudor** [comment]
Darn. I don't have AP-TAG, PS:M or TWTM. No portal mods (since desura or how it's called went bankrupt and got uninstalled, it held all my portal 1 mods)

**Comment by Endermage77** [comment]
Well, you can get those latter two for free off steam, although you'll actually have to purchase Aperture Tag for $4.99 or the equivalent.
